spec/requests/backable_spec.rb

```diff
@@ -31,7 +31,7 @@
     end
 
     it 'has a back link to the previous page' do
-      expect(response.body).to have_back_link("#{page_2}?back=true")
+      expect(response.body).to have_back_link("#{page_2}&back=true")
```

Comment:
This format looks wrong, shouldn't urls be
`domain?query_params#anchor`
I can't visualise how the locale will fit in this example

Reply:
Remember that all urls are now generated with a locale parameter.

`page_2` in this case is the full url including the locale param, so something like `/citizens/own_home?locale=en`.  leaving it as it was would have resultted in /citizens/own_home?locale=en?back=true` which is obviously wrong.